frontend/heir/mlir_emitter.py

```diff
@@ -446,7 +540,10 @@ def emit_assign(self, assign):
           self.forward_name_to_id(assign.target, name.strip("%"))
         return const_str
       case ir.Global():
-        self.globals_map[assign.target.name] = assign.value.name
+        self.globals_map[assign.target.name] = (
+            assign.value.name,
+            assign.value.value,
```

Comment:
Please add a comment at the definition

```
self.globals_map = {}
```

To explain what is in this map. Now that it's more than string->string it's not clear what `assign.value.value` actually refers to. What is the type of `global_` when the name is "bool"?